.github/workflows/tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
is there a need for ""

Reply:
I don't think so. You can open a PR to this brach without the quotes and if it works, I will merge it

Reply:
Without the quotes the action won't work if I recall correctly.